### PR TITLE
Add CSS.GG — 700+ Pure CSS, SVG & Figma UI Icons

### PR DESCRIPTION
**Pull request by astrit**
## Description
Open-source CSS, SVG and Figma UI Icons Available in SVG Sprite, styled-components, NPM & API

## Type of change

- [x] New extension

## Screencast

https://user-images.githubusercontent.com/2398447/142507000-369d2492-57b6-4d84-ab1c-1c369f981ed2.mp4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I checked that extension's icon has size of `512x512`

### CODE REVIEW

File: extensions/css-gg/src/css-gg.tsx [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+              url={"https://css.gg/" + name}
+              icon={{
+                source: {
+                  light: apiPathLight + "globe-alt.png",
+                  dark: apiPathDark + "globe-alt.png",
+                },
+              }}
+            />
+          </ActionPanel.Section>
+          <ActionPanel.Section title="Copy as inline">
```
Comment (pernielsentikaer) [inline extensions/css-gg/src/css-gg.tsx:117]:
```suggestion
          <ActionPanel.Section title="Copy as Inline">
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @astrit 👋 

Thank you for this contribution, I left a few suggestions and comments that we need to address.

- Icons in ActionPanel should be added in assets folder

File: extensions/css-gg/package.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "description": "Open-source CSS, SVG and Figma UI Icons Available in SVG Sprite, styled-components, NPM & API",
+  "icon": "css-gg.png",
+  "author": "astrit",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "css-gg",
+      "title": "CSS.GG",
+      "subtitle": "700+ CSS & SVG Icons ",
+      "description": "Open-source CSS, SVG and Figma UI Icons Available in SVG Sprite, styled-components, NPM & API",
```
[the remark this answers is not in the document]
Comment (astrit) [reply extensions/css-gg/package.json]: @pernielsentikaer I did update these.

**Comment by astrit** [comment]
Hi @pernielsentikaer , 

Thank you very much for the feedback. 
I did update the Command fields as requested, please check it and let me know if all good. 

On regards to: 
* Icons in ActionPanel should be added in assets folder

Can I keep it as it is with the icons coming from the `cdn*` , reason why I did not add on assets folder and get them directly from `url` was not to make the extension itself heavy. 

As initially I thought id leave all PNG versions on the extension Light/Dark mode but those are 1,408 PNG and that is why I hosted on Vercel everything. 

Otherwise if absolutely necessary I can download them and add them to the `asset` directory.

**Review by pernielsentikaer — APPROVED** [review]
Hi @astrit 👋 

I'm going to accept it and then my colleague will do a last check before merging 💪

**Pull request merged**